storybook/CMakeLists.txt

```diff
@@ -26,6 +40,7 @@ find_package(Qt${QT_VERSION_MAJOR} COMPONENTS Core Gui Quick QuickControls2 Test
 set(STATUSQ_BUILD_SANDBOX OFF)
 set(STATUSQ_BUILD_SANITY_CHECKER OFF)
 set(STATUSQ_BUILD_TESTS OFF)
+set(STATUSQ_STATIC_LIB ON)
```

Comment:
Why? The dll is not enough?

Reply:
dllimport issues. We'd need to export these functions or use the already exported c functions.

Do you see any advantage for the dynamic lib here?

Reply:
Not a specific one, was just wondering about the sudden addition of the static lib.

Reply:
Another case in point; for Windows and DOtherSide, we use the exact opposite scenario: dynamic lib for Windows, static for other OSs

Reply:
Yeah.. not sure either myself :)

Reply:
fwiw, building `DOtherSide` is _usually_ best done with `{.compile.}` in nim - this gets rid of most `.dll`/`.a` problems and reduces the amount of moving parts in the build system overall, no matter the platform.

On windows, you have to be extra careful because memory deallocations must be performed from within the DLL that allocated the memory - eliminating dll:s is an easy way avoid this problem altogether.